frontend/package.json

```diff
@@ -112,7 +116,7 @@
     "start": "cross-env REACT_APP_HEADLAMP_BACKEND_TOKEN=headlamp vite",
     "preview": "vite preview",
     "prebuild": "npm run make-version",
-    "build": "cross-env PUBLIC_URL=./ NODE_OPTIONS=--max-old-space-size=8096 vite build && npx shx rm -f build/frontend/index.baseUrl.html",
+    "build": "cross-env PUBLIC_URL=/ NODE_OPTIONS=--max-old-space-size=8096 vite build && npx shx rm -f build/frontend/index.baseUrl.html",
```

Comment:
This change wasn't mentioned in a commit message. I'm pretty sure this is depended on for baseURL functionality... does this still work?

Reply:
not sure why this was added, reverted